tests/functional/compile/fixtures.py

```diff
@@ -32,6 +32,16 @@
 }} as fun
 """
 
+with_recursive_model_sql = """
+{{ config(materialized = 'ephemeral') }}
+WITH RECURSIVE t(n) AS (
+    SELECT * FROM {{ ref('first_ephemeral_model') }}
+  UNION ALL
+    SELECT n+1 FROM t WHERE n < 100
+)
+SELECT sum(n) FROM t;
```

Comment:
Could you convert these to lowercase to match the style in the rest of this file?